change backend api endpoint

rasheedamir opened the pull request:
change backend api endpoint

rasheedamir committed:
change backend api endpoint

stakater-user commented:
@rasheedamir Image is available for testing. `docker pull docker.io/stakater/react-sockjs-keycloak:node-9.6.1-SNAPSHOT-PR-2-1`

rasheedamir commented:
Got this error:

```
Unhandled Rejection (SyntaxError): The URL's scheme must be either 'http:' or 'https:'. 'wss:' is not allowed.
new r
https://ws-fe.stakater-demo.lab187.k8syard.com/js/sockjs.min.js:2:9482
(anonymous function)
src/app/src/pages/private.js:113
  110 | store.dispatch(userSuccess(userInfo));
  111 | 
  112 | // TODO: shouldn't be hardcoded
> 113 | var socket = new window.SockJS('wss://ws-be.stakater-demo.lab187.k8syard.com/ws');
  114 | stompClient = window.Stomp.over(socket);
  115 | stompClient.connect({ Authorization: "Bearer " + keycloak.token},  (frame) => {
  116 |   console.log('Connected: ' + frame);
View compiled
This screen is visible only in development. It will not appear if the app crashes in production.
Open your browser’s developer console to further inspect this error.
```

Will change it to https and then see!

rasheedamir committed:
change to https from wss

sockjs throws exception if I set it to wss; it must be https; and then I think it will switch to websockets

stakater-user commented:
@rasheedamir Image is available for testing. `docker pull docker.io/stakater/react-sockjs-keycloak:node-9.6.1-SNAPSHOT-PR-2-2`

stakater-user commented:
@rasheedamir Image is available for testing. `docker pull docker.io/stakater/react-sockjs-keycloak:node-9.6.1-SNAPSHOT-PR-2-1`

stakater-user commented:
@rasheedamir Image is available for testing. `docker pull docker.release.stakater.com:443/stakater/react-sockjs-keycloak:node-9.6.1-SNAPSHOT-PR-2-1`
